### ISSUE TITLE
Unable to create index with more than one index type

### ISSUE THREAD

**Issue by rvijayakumar82** [jprante/elasticsearch-jdbc#651]
I am trying to create the index via the jdbc importer settings. I can only create one index type using this even though all the types are defined in "type_mapping" property. Is there a way that i can create more than one index type using jdbc importer?

**Comment by jprante** [comment]
No, this is a limitation in existing versions. Defining the index settings/mapping will be improved in future versions.
